bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -130,16 +172,17 @@ case class NameCatalogQuery(search: String, catalog: CatalogName) extends Catalo
 }
 
 object CatalogQuery {
-  // Useful constructors
-  def apply(id: Int, base: Coordinates, radiusConstraint: RadiusConstraint, magnitudeConstraints: List[MagnitudeConstraints], catalog: CatalogName):CatalogQuery = ConeSearchCatalogQuery(id.some, base, radiusConstraint, magnitudeConstraints, catalog)
-
-  def apply(base: Coordinates, radiusConstraint: RadiusConstraint, magnitudeConstraints: List[MagnitudeConstraints], catalog: CatalogName):CatalogQuery = ConeSearchCatalogQuery(None, base, radiusConstraint, magnitudeConstraints, catalog)
-
-  def apply(base: Coordinates, radiusConstraint: RadiusConstraint, magnitudeConstraints: MagnitudeConstraints, catalog: CatalogName):CatalogQuery = ConeSearchCatalogQuery(None, base, radiusConstraint, List(magnitudeConstraints), catalog)
 
-  def apply(base: Coordinates, radiusConstraint: RadiusConstraint, catalog: CatalogName):CatalogQuery = ConeSearchCatalogQuery(None, base, radiusConstraint, Nil, catalog)
+  def coneSearch(
+    c: edu.gemini.spModel.core.Coordinates,
+    r: RadiusConstraint,
+    m: MagnitudeConstraints,
+    n: CatalogName
+  ): CatalogQuery =
+    ConeSearchCatalogQuery(None, c, r, List(m), n)
 
-  def apply(search: String):CatalogQuery = NameCatalogQuery(search, CatalogName.SIMBAD)
+  def nameSearch(search: String): CatalogQuery =
+    NameCatalogQuery(search, CatalogName.SIMBAD)
```

Comment:
These are the two most used `CatalogQuery` constructors so I named them appropriately and eliminated the others (though you can still get the same things constructed via the `ConeSearchCatalogQuery.apply`).